### ISSUE TITLE
"pstoedit" is not included in $PATH

### ISSUE THREAD

**Issue by StefanPofahl** [octave-snap/octave-snap#96]
https://savannah.gnu.org/bugs/?61469
Please could you add "pstoedit" to the Octave Snap?

**Comment by mtmiller** [comment]
I can take a look at adding this. It seems to be a very rarely used code path and no one has reported it missing yet, that's the only reason it's not included.

Can you give an example that uses this program so I can test that it is working and found by Octave?

**Comment by StefanPofahl** [comment]
Hi Mike,
plotting nice pdf in custom size, is under Octave v6.4 difficult to achieve.
I made the experience that using the print() command is sometimes still 
a good choice under Octave v6.4.
I have removed the Octave v6.4 because of it limitations.
I attach a script that may help to track down the plotting issues on the 
different graphics_toolkit-options.
If further input from my side is needed please let me know and I reinstall Octave v6.4
and try to reproduce the error.
Regards,
Stefan

[example_isolines1.m.txt](https://github.com/octave-snap/octave-snap/files/8527663/example_isolines1.m.txt)

**Comment by mtmiller** [comment]
Ok, thanks. I changed the file names to run on my system and got your script to work.

It really comes down to a much simpler example:

```
peaks
print /tmp/plot.emf
```

It seems to require using the EMF format, which I gather is a format that some Windows users use but no one else uses.

If I comment out the lines dealing with EMF, pstoedit is not needed. no errors or warnings are issued, and all other expected files are generated correctly.

I'll use this test case and try adding pstoedit and make sure it gets called correctly.

**Comment by StefanPofahl** [comment]
Yes, that is true, the *.emf format is quite handy in the MS-windows world.
Thanks for your effort :-)
